api/src/main/java/com/github/retrooper/packetevents/wrapper/play/server/WrapperPlayServerDisconnect.java

```diff
@@ -48,26 +46,24 @@ public WrapperPlayServerDisconnect(String reasonJson) {
 
     @Override
     public void read() {
-        int maxMessageLength = serverVersion.isNewerThanOrEquals(ServerVersion.V_1_13) ? MODERN_MESSAGE_LENGTH : LEGACY_MESSAGE_LENGTH;
-        reasonJson = readString(maxMessageLength);
+        reasonJson = readString(getMaxMessageLength());
```

Comment:
does getMaxMessageLength do the same thing as the check we had here previously?

Reply:
yes